### ISSUE TITLE
.scatter() does not show scatter points on the map

### ISSUE THREAD

**Issue by ZED711** [vgm64/gmplot#106]
Hello everyone, 

I have an issue regarding scatter(). I will attach several prints of the code and the output. The code invokes the function full_map which draws markers, scatter points, activates api key and draws the html. The function **scatter** invokes .scatter method (where I have the issue). 
![print1](https://user-images.githubusercontent.com/41108785/63586675-d1adb080-c599-11e9-8cd1-d12529190132.PNG)
![print2](https://user-images.githubusercontent.com/41108785/63586676-d1adb080-c599-11e9-8ab3-f0a1c8348e0a.PNG)
![print3](https://user-images.githubusercontent.com/41108785/63586677-d2464700-c599-11e9-9909-4d4d5e93e3cb.PNG)
![print4](https://user-images.githubusercontent.com/41108785/63586678-d2464700-c599-11e9-8f2f-035ed5d13821.PNG)
![print5](https://user-images.githubusercontent.com/41108785/63586679-d2464700-c599-11e9-9192-2735e065da8d.PNG)

Basically, I create the gmap object with gmplot.GoogleMapPlotter() with the coordinates of the city Cascais in Portugal. 

After that, I am using gmap.marker to mark those purple markers that are seen on the map. 
The issue comes after, when using the function **scatter**, where I call the .scatter() method of gmplot (with marker = False). I use as input correct latitude and longitude values of a data frame data (I printed the values so that you can see exactly the data). However, the points are not scattered.

What am I doing wrong?

**Comment by ZED711** [comment]
Solved.
